### PR TITLE
Fix convergence detection bug in advi

### PR DESCRIPTION
**Pull request by aseyboldt**
Two changes:
- Do not check for convergence after the first iteration in advi. I've seen a model where it reported convergence in the first iteration.
- Close the tqdm progress bar

### CODE REVIEW

File: pymc3/variational/advi.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def advi(vars=None, start=None, model=None, n=5000, accurate_elbo=False,
                 avg_elbo = elbos[i - n // 10:i].mean()
                 progress.set_description('Average ELBO = {:,.5g}'.format(avg_elbo))
 
-            if i % eval_elbo == 0:
+            if i > 0 and i % eval_elbo == 0:
                 elbo_prev = elbo_current
                 elbo_current = elbos[i]
                 delta_elbo = abs((elbo_current - elbo_prev) / elbo_prev)
```
Comment (aseyboldt) [inline pymc3/variational/advi.py]: @lbollar Why is `circ_buff.append` inside the `i % eval_elbo == 0` branch? Shouldn't this be executed at every iteration?

**Comment by lbollar** [comment]
The append was only periodically added and checked to duplicate
implementation in Stan. If you wanted every iteration to append and check
wouldn't evaluation window set to 1 accomplish this?

**Comment by aseyboldt** [comment]
You are right, I wrote this before I thought it through, sorry.

**Comment by twiecki** [comment]
@aseyboldt Do you want to make more changes here?

**Comment by aseyboldt** [comment]
No, I think it's fine now.

**Pull request merged**